core/build.gradle.kts

```diff
@@ -97,8 +97,8 @@ configure<BlossomExtension> {
     replaceToken("\${repository}", info.repository, mainFile)
 }
 
-fun Project.buildNumber(): Int =
-    (System.getenv("GITHUB_RUN_NUMBER") ?: jenkinsBuildNumber())?.let { Integer.parseInt(it) } ?: -1
+// -1 as a fallback for local builds
+fun buildNumber(): Int = Integer.parseInt(System.getenv("GITHUB_RUN_NUMBER") ?: "-1")
```

Comment:
`System.getenv("GITHUB_RUN_NUMBER")?.let { Integer.parseInt(it) } ?: -1` might be a bit cleaner so we don't have to integer parse a constant string.